src/components/Identity/identityUtil.js

```diff
@@ -0,0 +1,61 @@
+import { AUTH_STATES } from "./constants";
+import isObject from "../../utils/isObject";
+
+const validateCustomerIDs = customerIDs => {
+  if (!isObject(customerIDs)) {
+    throw new Error(
+      "Invalid customer ID format. Each namespace should be an object"
+    );
+  }
+  Object.keys(customerIDs).forEach(customerID => {
+    if (!isObject(customerIDs[customerID])) {
+      throw new Error(
+        "Invalid customer ID format. Each namespace should be an object"
+      );
+    }
+    if (!customerIDs[customerID].id) {
+      throw new Error(
+        "Invalid customer ID format. Each namespace object should have an id"
+      );
+    }
+  });
+};
+const normalizeCustomerIDs = customerIDs => {
+  const normalized = {};
+  Object.keys(customerIDs).forEach(customerID => {
+    const currentID = customerIDs[customerID];
+    normalized[customerID] = { ...currentID };
```

Comment:
Does `{ ...currentID }` throw a `Symbol` exception in `IE 11`?

Reply:
It is working on IE11, tests are passing